### DISCUSSION TITLE
personalization

### DISCUSSION THREAD

**Discussion by Rachizoy** [Swordfish90/Lemuroid#974] [Ideas]
First I would like to apologize because I am using Google Translate to write this idea so maybe there is something written wrong, I'm sorry for that. Well, to be brief I think it would be cool to add an achievement system to your emulator with compatibility with RetroAchievements I would also love the possibility of changing the game covers well at least I didn't find this functionality on Android this was just some ideas I had for better customization thanks for reading.

**Comment by x64x2** [comment]
No
